src/test/java/com/auth0/client/auth/AuthAPITest.java

```diff
@@ -622,6 +630,15 @@ public void shouldCreateLogInWithPasswordGrantRequest() throws Exception {
         assertThat(response.getExpiresIn(), is(notNullValue()));
     }
 
+    @Test
+    public void passwordGrantRequestRequiresSecret() throws Exception {
+        AuthAPI authAPI = AuthAPI.newBuilder("domain", "clientId").build();
```

Comment:
NIT: We can refactor this outside so that we can reuse it for all requests that doesn't need secret

Reply:
yes, extracted to reuse.